pf/tests/internal/testprovider/testbridge_resource_compopt.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2016-2023, Pulumi Corporation.
```

Comment:
This file is called `testbridge_resource_compopt.go` If `compopt` is "component options", can we just say that? Otherwise, can we say what it is?